quix-frontend/client/src/lib/runner/services/autocomplete/high_light_and_score.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  createMatchMask,
+  findAllIndexOf,
+} from './autocomplete-utils';
+import { findRelevantPartOfPrefix } from "../../../sql-autocomplete/adapter/sql-autocomplete-adapter-utills";
+import { ICompleterItem as AceCompletion } from '../../../code-editor/services/code-editor-completer';
+import { QueryContext } from '../../../sql-autocomplete/sql-context-evaluator';
+
+
+export function enrichCompletionItemAfterDotObject(all: AceCompletion[]): AceCompletion[] {
+  const indexes = [0];
+  const perfectScore = 10000;
+  for (const completionItem of all) {
+    completionItem.matchMask = createMatchMask(indexes, 0);
+    completionItem.score = perfectScore;
```

Comment:
because there is no need to calculate...
they will all have the perfect score and no highlight.

do you think it will lead to better understanding ?